crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -506,6 +508,48 @@ impl<'a> SystemParamFetch<'a> for CommandQueue {
     }
 }
 
+impl<'a> SystemParam for ScheduleCommands<'a> {
+    type Fetch = ScheduleCommandQueue;
+}
+
+// SAFE: ScheduleCommands only accesses internal state
```

Comment:
```suggestion
// SAFE: ScheduleCommands have no effect until the next hard sync point
```

Reply:
@alice-i-cecile original comment is correct?

Reply:
The original comment is true, but not very clear. On a second look, I think that specifying what "internal" means is the better direction to go.

Reply:
This is no longer needed and has been removed.